docs/wes-qc-hail.md

```diff
@@ -52,6 +96,17 @@ Load VCFs into Hail and save as a Hail MatrixTable
 spark-submit 1-import_data/1-import_gatk_vcfs_to_hail.py
 ```
 
+Create the 1000G population prediction resource set.
+This resource set is required for the super-population prediction on the population PCA step.
+
+```shell
+spark-submit 1-import_data/1-import_gatk_vcfs_to_hail.py --all
+```
+
+You need to create this resource set only once.
+Then you can reuse it with any data cohort.
+
+
```

Comment:
need to add docs for step 1/4

Reply:
These are the docs for step 1/4. I incorrectly pasted the script's name. It's fixed now.